src/client/containers/SignUp/index.js

```diff
@@ -21,11 +30,26 @@ export default function SignUpContainer() {
       alert("Passwords doesn't match");
       return;
     }
-    const isSignedUp = await signUp({ email, password });
-    setIsLoading(false);
-    if (isSignedUp) setIsSuccessful(true);
+    const firebaseUser = await signUp({ email, password });
+    if (firebaseUser) {
+      await fetch('/api/users', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          firebaseToken: firebaseUser.user.uid,
+          fullName,
+          position,
+          linkedin,
+          github,
+          website,
+        }),
+      });
+      setIsLoading(false);
+      history.push('/profile');
+    }
```

Comment:
I think this is nice but it would be great to maybe wrap this API call in a `try{}catch{}` block because even if the API call fails here it will try to redirect you to `/profile` which I guess we only want to do if it is successful? :) 

But it should it work so minor thing :)

Reply:
i actually removed the try catch block from the template as it wasn´t needed. It had a successful check component that i deleted since it would only redirect you to profile if it is successfull. if not it will have page not found..